packages/api-nest/test/content-extraction.e2e-spec.ts

```
@@ -0,0 +1,174 @@
+import { INestApplication } from '@nestjs/common'
+import { Queue } from 'bullmq'
+import request from 'supertest'
+import { createE2EApp } from './helpers/create-e2e-app'
+import { FactoryRegistry } from './factories/base.factory'
+import { QUEUE_NAMES } from '../src/queue/queue.constants'
+import { LibraryItemState } from '../src/library/entities/library-item.entity'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)